include/openssl/ocsp.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2015-2021 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+
+#ifndef AWSLC_OCSP_H
+#define AWSLC_OCSP_H
+
+#pragma once
```

Comment:
I guess this is redundant when using the guard `AWSLC_OCSP_H` above. Prefer `pragma`'s but not used in AWS-LC.